_episodes/16-parallel.md

```diff
@@ -1,28 +1,36 @@
 ---
 title: "Running a parallel job"
 teaching: 30
-exercises: 30
+exercises: 60
 questions:
 - "How do we execute a task in parallel?"
+- "What benefits arise from parallel execution?"
+- "What are the limits of gains from execution in parallel?"
 objectives:
-- "Understand how to run a parallel job on a cluster."
+- "Construct a program that can execute in parallel."
+- "Prepare a job submission script for the parallel executable."
+- "Launch jobs with increasing degrees of parallel execution."
+- "Record and summarize the timing and accuracy of the results."
 keypoints:
-- "Parallelism is an important feature of HPC clusters."
-- "MPI parallelism is a common case."
+- "Parallel programming allows applications to take advantage of
+  parallel hardware; serial code will not 'just work.'"
+- "Distributed memory parallelism is a common case, using the Message
+  Passing Interface (MPI)."
 - "The queuing system facilitates executing parallel tasks."
+- "Performance improvements from parallel execution do not scale linearly."
 ---
 
 We now have the tools we need to run a multi-processor job. This is a very
-important aspect of HPC systems, as parallelism is one of the primary tools we
-have to improve the performance of computational tasks.
+important aspect of HPC systems, as parallelism is one of the primary tools
+we have to improve the performance of computational tasks.
 
 Our example implements a stochastic algorithm for estimating the value of
-&#960;, the ratio of the circumference to the diameter of a circle.
-The program generates a large number of random points on a 1&times;1 square
-centered on (&frac12;,&frac12;), and checks how many of these points fall
+π, the ratio of the circumference to the diameter of a circle.
```

Comment:
Aren't these rendered characters a potential source of editor pain?

Reply:
Yes, but after switching them all to HTML codes, the raw Markdown became much more difficult to understand! My suggestion for those with uncooperative editors -- and I know this sounds snarky, I mean it with tenderness & affection -- is to upgrade your terminal and/or editor, or use the Web interface.